src/librustdoc/html/static/rustdoc.css

```diff
@@ -1107,3 +1107,17 @@ h3.important {
 	left: -42px;
 	margin-top: 2px;
 }
+
+kbd {
+	display: inline-block;
+	padding: 3px 5px;
+	font: 11px "SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace;
+	line-height: 10px;
+	color: #444d56;
+	vertical-align: middle;
+	background-color: #fafbfc;
```

Comment:
Oh forgot this: please put all color rules into the `main.css`. The point is to make easier the add of multiple themes by just changing one CSS file instead of several.